apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/ScanToConnectView.kt

```diff
@@ -1,57 +1,51 @@
 package chat.simplex.common.views.newchat
```

Comment:
should be good to move to NewChatView, or do you want to do it in a separate pr to keep diff?

Reply:
I would prefer we both to have this code separated from other classes. Like ConnectPlan.{kt,swift}. What do you think?
- it's unrelated to this PR and to its idea
- it has some logic nobody really cares about
- it has no UI, just logic.

Let's just move it to separate file (in my case I can just rename it, in you case you can move the whole big part of code to the file ConnectPlan.swift).

I don't really like to mix such big parts of UI and logic into one SQLite-like file

Reply:
Ok, fine to rename too, or even keep as is, doesn't matter much. In swift it's slightly more tied to views with those ugly Alert and Sheet enums, but frankly I just don't want to bother moving second time

Reply:
Ah, ok, didn't think about how tied they are. Let me rename my file than and will keep it like this

Reply:
renamed